{
  final Exception ex=new Exception();
  try {
    testRollback(new TransactionOperationCallback(){
      @Override public Object performTransactionalOperation() throws Throwable {
        return annotationOnlyOnClassWithNoInterface.echo(ex);
      }
    }
,false);
    fail("Should have thrown Exception");
  }
 catch (  Exception ex2) {
    assertSame(ex,ex2);
  }
}
